.github/workflows/release.yml

```diff
@@ -0,0 +1,67 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - v*
+
+permissions:
+  contents: write
+
+jobs:
+  maybe_create_release:
+    runs-on: ubuntu-22.04
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - name: Check if release already exists
+        id: check_release
+        run: |
+          RELEASE_EXISTS=$(gh release view ${{ github.ref_name }} \
+                              --repo ${{ github.repository }} 2>&1 \
+                              | grep 'release not found' \
+                              || echo 'found')
+          echo "RELEASE_EXISTS=$RELEASE_EXISTS" >> $GITHUB_ENV
+
+      - name: Create release
+        if: env.RELEASE_EXISTS == 'release not found'
+        run: |
+          echo "Creating release..."
+          gh release create \
+            --repo ${{ github.repository }} \
+            --title ${{ github.ref_name }} \
+            ${{ github.ref_name }}
```

Comment:
Quick question: could we always run release create? I assume if we try to create a release for an existing ref, it is going to fail and we can just ignore it? If that's the case, then we can always try to create the release within each job. WDYT?

Reply:
I think it makes sense, then you avoid any intervention in order to remove the release 🚀.
I have updated it so it continues-on-error when creating the release.

Reply:
Example of run that continues-on-error https://github.com/gilacost/ex_doc/actions/runs/6829969263